resources/resinos.robot

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Documentation    This resource provides access to host OS.
+Library   Process
+Library   OperatingSystem
+Library   String
+Library   Collections
+
+*** Variables ***
+
+*** Keywords ***
+Set up loop device for "${path_to_image}"
+    ${result} =  Run Process    losetup -f -P ${path_to_image}    shell=yes
+    Process ${result}
+
+Find the last loop device that was assigned
+    ${result} =  Run Process    lsblk | grep 7: | tail -1 | awk -F: '{ print $2 }' | cut -d ' ' -f 1    shell=yes
+    Process ${result}
+    Should Not Be Empty     ${result.stdout}   msg="Couldn't find the loop device"
+    Set Suite Variable    ${LOOPDEVICE}    ${result.stdout}
+
+Mount "${path}" on "${mount_destination}"
+    ${result} =  Run Process    mount ${path} ${mount_destination}      shell=yes
+    Process ${result}
+    Directory Should Not Be Empty   ${mount_destination}    msg=Couldn't mount loop device to ${mount_destination}
```

Comment:
@horia-delicoti This seems wrong - We are verifying a successful mount by checking if the directory is empty. Process ${result} will already ensure that the mount was successful.